src/components/Common/FileDropZone.tsx

```diff
@@ -0,0 +1,75 @@
+import { memo, useCallback } from 'react';
+import { useDropzone } from 'react-dropzone';
+import styled from 'styled-components';
+
+import { prefecthCid } from '@helpers/prefetchCid';
+
+import ImagePreview from './ImagePreview';
+
+const SDropZone = styled.div`
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  max-width: 500px;
+  border: 4px dashed #d33079;
+  border-radius: 2px;
+  outline: none;
```

Comment:
Shouldn't you also set a height or min-height for this?!

Reply:
this is a draft PR, it's unfinished
these styles are temporary and will be changed